pkg/controller/utils/capability.go

```diff
@@ -190,9 +190,16 @@ func GetOpenAPISchemaFromTerraformComponentDefinition(configuration string) ([]b
 // GetTerraformConfigurationFromRemote gets Terraform Configuration(HCL)
 func GetTerraformConfigurationFromRemote(name, remoteURL, remotePath string) (string, error) {
 	tmpPath := filepath.Join("./tmp/terraform", name)
+	// Check if the directory exists. If yes, remove it.
```

Comment:
@FogDong Here are two modifications of this method. Please help check whether they are okay with you.